content/blog/test-edge-cases/index.md

```diff
@@ -0,0 +1,155 @@
+---
+{
+	title: "Test Content: Edge Cases",
+	description: "Look at all those edge cases!",
+	published: '2020-09-18',
```

Comment:
Let's set this to `1999` or something so that we can easily detect if it makes it into prod